internal/cmd/commands/accountscmd/accounts.gen.go

```diff
@@ -155,8 +155,15 @@ func (c *Command) Run(args []string) int {
 	}
 
 	client, err := c.Client()
+	if c.WrapperCleanupFunc != nil {
```

Comment:
Reviewers: this is representative of how the various generated files changed, so you can take a look at this and ignore the rest.

What's happening here is simply ensuring that _if_ a cleanup func is specified by a wrapper, we ensure that it is run at the end of the command.